bplist/bplist.py

```diff
@@ -271,3 +272,17 @@ def unplist(s):
     from Foundation import NSData, NSPropertyListSerialization
     d = NSData.dataWithBytes_length_(s, len(s))
     return NSPropertyListSerialization.propertyListWithData_options_format_error_(d, 0, None, None)
+
+if __name__ == "__main__":
```

Comment:
I guess that adds a nice `python -m bplist` use case. Could you update the docs accordingly, please?